templates/docs/patterns/grid/index.md

```diff
@@ -41,167 +52,47 @@ Vanilla has a responsive grid with the following columns and gutters:
 
 <br>
 
-- The page structure must be laid out using rows (`.row`)
-- All content must be within columns (`.col-*`)
+- The page structure must be laid out using rows (`.grid-row`)
+- All content must be within columns (`.grid-col-*`)
 - Only columns should be direct children of a row
 
-Layouts can be created combining rows with different number of columns to an ideal maximum of 4 columns per row. Each column containing text should span a minimum of 3 columns.
+Layouts can be created combining rows with different number of columns to an ideal maximum of 4 columns per row. Each column containing text should span a minimum of 2 columns.
 
 Read also: [Breakpoints](/docs/settings/breakpoint-settings)
 
-<div class="embedded-example"><a href="/docs/examples/patterns/grid/default/" class="js-example">
+<div class="embedded-example"><a href="/docs/examples/patterns/grid-8/default/" class="js-example">
     View example of the default grid
 </a></div>
 
-## Common patterns
```

Comment:
Please keep the Common patterns section, mentioning that they are work in progress and recommend use of existing ones in the meantime.

Reply:
Done